test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -154,7 +115,7 @@ public void GenerateStartTag_ReturnsNothingIfWhitespaceTagName()
             var output = tagHelperOutput.GenerateStartTag();
 
             // Assert
-            Assert.Empty(output);
+            Assert.Null(output);
```

Comment:
don't understand why `GenerateStartTag()` returns `null` at the moment.  fine if you change that method to return a `TagHelperContent` instance.